UIDependent/UIDependent/Realization/Extensions/CGFloat+Extensions.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CGFloat+Extensions.swift
+//  UIDependent
+//
+//  Created by Nikita Grishin on 10.01.23.
+//
+
+public extension CGFloat {
+    static let canonicalHeight: CGFloat = 812
```

Comment:
I think it would be nice to allow developers to override a canonical size if they need it.